integration/docker/Dockerfile-dev

```diff
@@ -101,29 +39,4 @@ RUN wget -qO /tmp/async-profiler-1.8.3-linux-x64.tar.gz "https://github.com/jvm-
     mv /opt/async-profiler-* /opt/async-profiler && \
     rm /tmp/async-profiler-1.8.3-linux-x64.tar.gz
 
-# Disable JVM DNS cache in both java8 and java11 (https://github.com/Alluxio/alluxio/pull/9452)
-RUN echo "networkaddress.cache.ttl=0" >> /usr/lib/jvm/java-1.8.0-openjdk/jre/lib/security/java.security
-RUN echo "networkaddress.cache.ttl=0" >> /usr/lib/jvm/java-11-openjdk/conf/security/java.security
-
-# Add the following for native libraries needed by rocksdb
-ENV LD_LIBRARY_PATH /lib64:${LD_LIBRARY_PATH}
-
-# If Alluxio user, group, gid, and uid aren't root|0, create the alluxio user and set file permissions accordingly
-RUN ./dockerfile-common.sh user-operation ${ALLUXIO_USERNAME} ${ALLUXIO_GROUP} ${ALLUXIO_UID} ${ALLUXIO_GID} centos
-
-# Docker 19.03+ required to expand variables in --chown argument
-# https://github.com/moby/buildkit/pull/926#issuecomment-503943557
-COPY --from=alluxio-extractor --chown=${ALLUXIO_USERNAME}:${ALLUXIO_GROUP} /opt /opt/
-COPY --chown=${ALLUXIO_USERNAME}:${ALLUXIO_GROUP} conf /opt/alluxio/conf/
-COPY --chown=${ALLUXIO_USERNAME}:${ALLUXIO_GROUP} entrypoint.sh /
-COPY --from=csi-dev /usr/local/bin/alluxio-csi /usr/local/bin/
-
-RUN ./dockerfile-common.sh enable-dynamic-user ${ENABLE_DYNAMIC_USER}
-
-USER ${ALLUXIO_UID}
-
-WORKDIR /opt/alluxio
-
-ENV PATH="/opt/alluxio/bin:${PATH}"
-
-ENTRYPOINT ["/entrypoint.sh"]
+USER ${ALLUXIO_USERNAME}
```

Comment:
Why `USER ${ALLUXIO_UID}` in Dockerfile but USERNAME here?

Reply:
Doesn't really matter in terms of functionality. I'll put UID instead to avoid confusion.